docs/setup_installation/gcp/getting_started.md

```diff
@@ -7,7 +7,8 @@ SageMaker and KubeFlow. This guide shows how to set up [managed.hopsworks.ai](ht
 ## Prerequisite
 
 To follow the instruction of this page you will need the following:
-- A GCP project in which the hopsworks cluster will be deployed.
+
+- A GCP project in which the hopsworks cluster will be deployed. Make sure to enable *Compute Engine API*, *Cloud Resource Manager API*, and *Identity and Access Management (IAM) API* on that project.
```

Comment:
Could we link to some GCP documentation or something like that that would explain how to enable all of that?